### ISSUE TITLE
Availability on Azure Gov

### ISSUE THREAD

**Issue by guythetechie** [Azure/static-web-apps#1019]
Static Web Apps doesn't seem to be available on Azure Government. Is this on the roadmap?

**Comment by kamikaz8** [comment]
I have a similar query and would appreciate any guidance from the Azure team.  Thank you!

**Comment by sulabh-msft** [comment]
@Reshmi-Sriram / @thomasgauvin,
   Can you please take up the query?

**Comment by Reshmi-Sriram** [comment]
Hey folks, 
As of now we do not have it on our roadmap to provide Static Web Apps availability on Gov clouds since it has not been a big ask. We'd love to see if more folks are interested in this (possibly by monitoring this issue upvotes), and meanwhile would be more than happy to help the customers in any way possible to help in their Static Web Apps journeys :)

**Comment by nadoylemsft** [comment]
@Reshmi-Sriram I am a CSA-E with MSFT. I can tell you my Cx has at least 30 apps that Static Web Apps would go great with. However, their rule (always has exceptions), as with most of US PubSec, is to not entertain new technologies until they are GA due to requirements such as ATO, etc.

**Comment by pierowman** [comment]
@Reshmi-Sriram I am a CSA with MSFT and I can tell you this would be a big win for my federal customers and they have asked for this feature, repeatedly.

**Comment by sandman-battelle** [comment]
I publish documentation for environments that contain CUI data in Azure Gov but have to publish those web sites in Azure Commercial to use SWA.  This causes me to receive a "finding" during every audit that I have to create a POAM for, then work through the exception process, not fun.  Having this would be a major win for my team.

Moreover, MSFT expressed to us (via our account team) their intent that services in commercial would be available in Gov "soon" (they said 30 days) after the feature goes GA in commercial.  "1 year later..." seems like still no movement.  Given that we pay a serious premium for Azure Government resources, why would you not want to provide us with the best experience and give us all the services of commercial that you can.  SWA is basically a convenience service around a webservices (or a simple web server) so I doubt there are any serious compliance hurdles to overcome.

TL;DR
YES, MSFT.  OF COURSE WE WANT THIS IN GOV!! ❤️

**Comment by sandman-battelle** [comment]
> [snip] since it has not been a big ask. We'd love to see if more folks are interested [snip]

Seriously?? We (the paying customers) need to rally a community effort for MSFT to rollout services to Gov already available to commercial.  Otherwise, you're like "meh, guess they don't really want it so let's do nothing."  C'mon team, how about a little proactive effort rather than pushing it back on your users to fight-the-fight. [feels bad]

**Comment by guythetechie** [comment]
SWAs benefit any team that wants easy deployment and wide distribution of, well, static web app content. Gov customers have plenty of such content for internal, national, and international consumption.

If a service is deemed useful enough to make it to Azure public cloud, you can bet that Gov customers would benefit too. I understand that Gov has compliance requirements, but I don't see how SWAs would violate those when other "open" services (App Services, Front Door, etc) don't.

**Comment by guythetechie** [comment]
Just think of all the public government sites you visit: federal .gov sites, state DMVs, park sites, benefits sites, etc. Those are all great candidates for Static Web Apps.

**Comment by edwardfward** [comment]
How do we show a big ask? Call Jason Zander?

**Comment by FelipeA** [comment]
I'm waiting Static Web Apps on Gov too.
Also, It's not available on Canada regions.

**Comment by blueboxes** [comment]
I would like this also

**Comment by CData-Jfarris** [comment]
We also need this capability. I wish there were a better way for us to request these features. Even if it was in one or two regions, it would be better than none at all.

**Comment by bhartman353** [comment]
Definitely would be nice since they can be used just for proxies.

**Comment by shahiddev** [comment]
+1 for Gov Support 🙏

**Comment by brnlmrry** [comment]
+1 for gov support. Having to ask for this is incredibly disappointing.

**Comment by KevPat** [comment]
You figure how much more secure this would be than having a small surface
attack area instead of an entire app service just to put some info out
there.

**Comment by Stawsh** [comment]
Well here it is 1 year+ later and as one new to Azure Gov, moving from Azure Commercial, I run into this limitation ... what to do with my web site currently hosted in an SWA in Azure Commercial.

**Comment by bmurray-napaco** [comment]
You may be able to cobble the pieces together yourself:
- Create an Azure Storage resource, enable the "Static website" feature
- Create a Front Door and CDN profile that serves off the static website
- Use an endpoint to configure a custom domain

I was able to get what I needed going that route. It would be nice if this was wrapped up in a single service though.

**Comment by Stawsh** [comment]
Thanks for the outline of steps if I want to use Azure Storage.  What I did instead, so far, was to set up an App Service within the same App Service plan where I have the back-end server that this front-end web app needs to make (HTTP POST, etc.) calls to.  Dunno working yet, but I have created those Azure Services.

**Comment by JamesWilsonProjects** [comment]
Do we have any updates on this? I have customers in GCC wanted to use SWA

**Comment by jeremytbrun** [comment]
+1 on wanting SWA in Azure Government!

**Comment by mpolutta** [comment]
Samesie = +1 for SWA in gov environment

**Comment by KevPat** [comment]
+2, been watching for so many years

**Comment by jvataj** [comment]
+3 for SWA in Azure Gov!

**Comment by PipeItToDevNull** [comment]
Found this today, I never imagined this could be an issue
